stl/debugger/STL.natvis

```diff
@@ -1955,4 +2043,149 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+      <Item Name="Width index">_Specs._Width</Item>
+      <Item Name="Precision index">_Specs._Precision</Item>
+      <Item Name="Width index">_Specs._Type</Item>
+      <Item Name="Precision index">_Specs._Alignment</Item>
+      <Item Name="Width index">_Specs._Sgn</Item>
+      <Item Name="Precision index">_Specs._Alt</Item>
+      <Item Name="Width index">_Specs._Localized</Item>
+      <Item Name="Precision index">_Specs._Leading_zero</Item>
+      <Item Name="Width index">_Specs._Fill_length</Item>
+      <Item Name="Precision index">_Specs._Fill</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::basic_format_parse_context&lt;*&gt;">
+    <DisplayString>{_Format_string}</DisplayString>
+    <Expand>
+      <Item Name="Arguments">_Num_args</Item>
+      <Item Name="Next arg id">_Next_arg_id</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::basic_format_args&lt;*&gt;">
+    <DisplayString>Num args:{_Num_args}</DisplayString>
+    <Expand>
+      <Item Name="Type">_Index_array->_Type_</Item>
+      <IndexListItems>
+        <Size>_Num_args</Size>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 1">
+          *(reinterpret_cast&lt;const int*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 2">
+          *(reinterpret_cast&lt;const unsigned int*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 3">
+          *(reinterpret_cast&lt;const long long*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 4">
+          *(reinterpret_cast&lt;const unsigned long long*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 5">
+          *(reinterpret_cast&lt;const bool*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 6">
+          *(reinterpret_cast&lt;const unsigned char*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 7">
+          *(reinterpret_cast&lt;const float*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 8">
+          *(reinterpret_cast&lt;const double*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 9">
+          *(reinterpret_cast&lt;const long double*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index)
+        </ValueNode>
+        <ValueNode Condition="_Index_array[$i]._Type_ == 10">
+          (void*)(*(reinterpret_cast&lt;const unsigned char*&gt;(_Index_array + _Num_args) + _Index_array[$i]._Index))

```

Comment:
Is this correct? I haven't tested it, but it's using the "read an `unsigned char`" expression used for `_Char_type`, and then casting it to `void*` - I don't think that'll work for `_Pointer_type`.